content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,64 @@
+
+## Central Cluster Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you can deploy your services to the mesh, like the diagram below:
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+From Istio 1.7, it is possible to run Istiod on a separate, dedicated cluster (the "central control plane cluster") as shown in the diagram below. With this new deployment model, mesh operators can work purely on the central control plane cluster when there is a need to upgrade or re-configure Istio. Platform/service owners work solely on the data plane cluster to deploy their applications and apply any Istio resources or configurations. They don't have direct access to the central control plane cluster to deploy anything there, neither can the central control plane cluster process any Istio resources in that cluster. If you are interested in exploring this, you can follow the [central istiod single cluster step by step guide](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps).
+
+{{< image width="100%"
+    link="single-cluster-central-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in a central control plane cluster"
+    >}}
+
+The diagram above shows a single cluster as the data plane for an Istio mesh. However, you can expand this deployment model to manage multiple Istio meshes from a central control plane cluster that runs multiple Istiod control planes:
```

Comment:
I think you should try to be clear about your audience. Here you're talking to the mesh operator, and telling them that they can run multiple control plane instances in a single cluster.

It may be worthwhile to really target this blog post towards either mesh operators or mesh admins, or split into sections, one for each. Otherwise it is confusing who you're talking to and what you're trying to tell them.

For example, to the mesh operator, you want them to know:
- They can run the control plane in their own clusters, and provide it as a service to mesh admins.
- They can manage the lifecycle of the control plane (?)
- They can run multiple control planes in a single cluster.
- Mesh admins don't need access to their cluster.
- They can use their own istio mesh to manage their control plane clusters, using ingress to control access.

For mesh admins:
- They can use a control plane as a service provided by a mesh operator
- They can still configure mesh-wide settings using mesh config
- They can control when they upgrade to a new control plane version (?)

Reply:
Thanks, this is good feedback, I have been thinking to separate the roles in the diagram as well.  I'll rework on the split.

Reply:
Thanks - Clarified!  PTAL.  I left your two question bullets out as that is out of scope for this blog.